gradle-module-plugin/src/functionalTest/kotlin/io/ia/sdk/gradle/modl/task/WriteModuleXmlTest.kt

```diff
@@ -188,13 +217,26 @@ class WriteModuleXmlTest : BaseTest() {
     private fun generateXml(
         dirName: String,
         replacements: Map<String, String> = mapOf(),
+        dependencies: Map<String, String> = mapOf(),
         dumpBuildScript: Boolean = false,
     ): String {
         val projectDir = generateModule(
             tempFolder.newFolder(dirName),
             replacements,
         )
 
+        dependencies.forEach { (scopes, dependency) ->
```

Comment:
Added the ability to included additional dependencies in the `build.gradle` files of the generated scope subprojects.